gpib_server.py

```diff
@@ -145,6 +141,23 @@ def getDevice(self, c):
         instr = self.devices[c['addr']]
         return instr
 
+    def asynchronous_wait(self, seconds, result=None):
```

Comment:
@maffoo , yep, that works.  Superior.  See: https://github.com/martinisgroup/servers/commit/094710dc45d5c8a8bea7cc5addb0c9ac33f32694